cmd/gtk/model_wallet.go

```diff
@@ -48,12 +54,23 @@ func (model *walletModel) rebuildModel() {
 			balanceStr := util.ChangeToString(balance)
 			stakeStr := util.ChangeToString(stake)
 
+			var score string
+
+			valAddr, err := crypto.AddressFromString(info.Address)
+			if err == nil {
+				val := model.node.State().ValidatorByAddress(valAddr)
```

Comment:
val itself has `availibility_score`. right?

Reply:
ni, validator type does not contain an availability_score.